opm/output/eclipse/EclipseWriter.cpp

```diff
@@ -548,10 +530,10 @@ void EclipseWriter::writeInit( const NNC& nnc ) {
         fortio.writeKeyword("PERMZ", data);
     }
 
-    if( !nnc.hasNNC() ) return;
+    if( !this->impl->nnc.hasNNC() ) return;
 
     std::vector<double> tran;
-    for( NNCdata nd : nnc.nncdata() ) {
+    for( NNCdata nd : this->impl->nnc.nncdata() ) {
```

Comment:
Maybe, but I sincerely doubt it will be faster. Since it's more-or-less immediately pushed back into the vector the temporary might be elided (and if not it's in super hot cache anyway). It's also 24 bytes so very unlikely to make a difference.

That being said, the reallocation thing is worse. I'm pretty sure most of this will be adapted to a heavy mapping bias once we get Windows CI to work and this file as a whole gets some love. So far it's mostly been a port from the old system modulo its opm-core dependencies. Cleanups are coming though.